.github/workflows/devel-doc.yml

```diff
@@ -0,0 +1,43 @@
+---
+# This is deploying the latest commits on devel to aristanetworks/avd-docs main branch
+name: Deploy devel documentation to aristanetworks/avd-docs main
+
+on:
+  push:
+    branches:
+      - devel
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
We should not cancel in progress flows, since it may leave cloudvision in a weird state.

Reply:
Can you please elaborate? This has nothing to do with cloudvision I think

Reply:
wrong pr. sorry.